FileCloner/.editorconfig

```diff
@@ -0,0 +1,313 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
What is the need of adding this inside FileCloner module ?, we have at the project level right ?

Reply:
But it was not analyzing the FileCloner module, for some reason, it was not showing any suggestions/corrections